hack/jenkins/verify-dockerized.sh

```diff
@@ -29,6 +29,17 @@ retry() {
 # This script is intended to be run from kubekins-test container with a
 # kubernetes repo mapped in. See k8s.io/test-infra/scenarios/kubernetes_verify.py
 
+REMOTE="$(git remote)"
+
+if [[ ${REMOTE} == *"bootstrap-upstream"* ]]; then
+	echo "from kubernetes_verify.py, has remote"
+else
+	echo "from podutils, adding remote..."
+	git remote add bootstrap-upstream https://github.com/kubernetes/kubernetes.git
```

Comment:
I don't think we need to call this `bootstrap-upstream` - just `upstream` should be fine.